### ISSUE TITLE
508-defect-2 [HEADINGS, COGNITION]: "On this page" heading SHOULD be an H2; Heading levels SHOULD only increase by one

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#13546]
#  [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

**Feedback framework**
- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

The On this page component heading **should** be an H2, and is currently an H3 on this page. For consistency throughout the website, screen reader users will come to expect this component to be a heading level 2.

## Point of Contact

**VFS Point of Contact:** Jennifer

## Acceptance Criteria

As a screen reader user, I want to navigate the hierarchy of the page content consistently using heading levels to save time and frustration.

## Environment

- Operating System: all
- Browser: all
- Screenreading device: any
- Server destination: staging

## Steps to Recreate

1. Enter https://staging.va.gov/manage-va-debt/your-debt/ and please use Greg Anderson (user 1).
2. Inspect the code for the On this page heading
3. Verify that it is an H3

### Sample 

**Current code**
```html
<h3>On this page</h3>
```

**Recommended code**
```html
<h2 class="vads-u-margin-bottom--2 vads-u-font-size--lg" id="on-this-page">On this page</h2>
```

Note, this could also use the `.vads-u-font-size--h3` class; they're the same as indicated in the [VA font-size utility](https://design.va.gov/utilities/font-size).

## WCAG or Vendor Guidance (optional)

* [axe-core 3.4 - Heading levels should only increase by one](https://dequeuniversity.com/rules/axe/3.4/heading-order)

### Current implementation of On this page in Debt Letters MVP

![Screen Shot 2020-09-14 at 5 40 06 PM](https://user-images.githubusercontent.com/57469/93141522-53471380-f6b2-11ea-8f36-06c207ee09c3.png)

### Correct On this page structure shown on Pittsburgh VAMC

![Screen Shot 2020-09-14 at 5 39 43 PM](https://user-images.githubusercontent.com/57469/93141455-33afeb00-f6b2-11ea-8230-9fbe3f8bdabf.png)

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

**Comment by bradl3yC** [comment]
@jenstrickland please validate

**Comment by jenstrickland** [comment]
@bradl3yC Is there a different url I should use? `https://staging.va.gov/manage-va-debt/your-debt/` returns a 404.

**Comment by jenstrickland** [comment]
Validated 🎉
